alpha/scripts/compatCheck.php

```diff
@@ -766,6 +771,11 @@ function testAction($ipAddress, $fullActionName, $parsedParams, $uri, $postParam
 			return;
 		}
 
+		if( !trim($resultNew) && $resultOld )
```

Comment:
why not trim the old also, to make sure that we raise this error only when old is not empty and new is emptry